server/services/parcelActionsService.js

```diff
@@ -1,7 +1,24 @@
-const actions = require('../../data/actions.json')
+const actionsService = require('./actionsService')
+const parcelService = require('./parcelService')
+const rulesEngine = require('../rules-engine')
 
 module.exports = {
   get: async function (parcelRef) {
-    return actions
+    const actions = actionsService.get()
```

Comment:
The actionService has an `async` method signature, but an `await` is not being used when the function is called.

Causing an error when calling the endpoint, i.e. a get on `http://localhost:3002/parcels/SD74445738/actions`

Reply:
Thanks for that. Can't believe I missed that. Fixed now.